starboard/extension/player_configuration.h

```diff
@@ -0,0 +1,51 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef STARBOARD_EXTENSION_PLAYER_CONFIGURATION_H_
+#define STARBOARD_EXTENSION_PLAYER_CONFIGURATION_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define kCobaltExtensionPlayerConfigurationName \
```

Comment:
Should this be `kStarboardExtensionPlayerConfigurationName `,  as we have document as follows: https://developers.google.com/youtube/cobalt/docs/codelabs/starboard_extensions/codelab.

Reply:
Seconded, let's call them "Starboard extensions" everywhere

Reply:
Updated the name.